src/peakWatcher.cpp

```diff
@@ -50,6 +52,7 @@ T& varGet(V& var) {
 class Watcher;
 bool isWatchingAnything();
 void stop();
+bool isPaused{false};
```

Comment:
I'm sure I'm missing something, but I'm not quite following why we need isPaused when we didn't before.

Reply:
It's purely to keep track of paused state when switching tabs. Before adding isPaused, switching from a tab with peak watcher disabled to one with peak watcher enabled would report peak watcher paused because the timer was disabled.

Reply:
Ah. I was thinking you'd just start the timer if Peak Watcher is enabled in the tab you're switching to, but that would potentially cause you to lose the pause state since you can't tell whether it was disabled or paused in the previous tab.